.github/workflows/test-fice.yml

```diff
@@ -47,21 +47,26 @@ jobs:
         run: |
           micromamba info
           micromamba list
+          which python
+          python -m pip install scipy==1.9.3
 
       # We install fenics_ice and tlm_adjoint as developing modules
       - name: Install repositories as dev modules
+        shell: bash -l {0}
         run: |
           ls $GITHUB_WORKSPACE
           cd $GITHUB_WORKSPACE/fenics_ice
           pip install -e .
           cd $GITHUB_WORKSPACE/tlm_adjoint
           pip install -e .
+          micromamba info
 
       # Test fenics ice!
       - name: Test fice
         shell: bash -l {0}
         run: |
           cd $GITHUB_WORKSPACE/fenics_ice/
+          which python
```

Comment:
This is nice to leave it here ... as it prints out what python version we are using and indicate that is the right environment... I will leave it for futher debugging